keras_cv/models/classification/video_classifier.py

```diff
@@ -0,0 +1,149 @@
+# Copyright 2024 The KerasCV Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Video classifier model using pooling and dense layers."""
+
+import copy
+
+from keras_cv.api_export import keras_cv_export
+from keras_cv.backend import keras
+from keras_cv.models.backbones.backbone_presets import backbone_presets
+from keras_cv.models.backbones.backbone_presets import (
+    backbone_presets_with_weights,
+)
+from keras_cv.models.classification.video_classifier_presets import (
+    classifier_presets,
+)
+from keras_cv.models.task import Task
+from keras_cv.utils.python_utils import classproperty
+
+
+@keras_cv_export(
+    [
+        "keras_cv.models.VideoClassifier",
+        "keras_cv.models.classification.VideoClassifier",
+    ]
+)
+class VideoClassifier(Task):
```

Comment:
This also needs to go in the `keras_cv.models` `__init__.py` file in order to be present in the public API.

Reply:
this is already import in the `keras_cv.models.__init__.py` file, LOC 199, isn't it?

Reply:
Ah, missed the import, thanks!